deps/outcome/outcome/outcome.hpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_OUTCOME_HPP
+#define KAGOME_OUTCOME_HPP
+
+#include <outcome/outcome-external.hpp>
+#include <outcome/outcome-register.hpp>
+
+namespace outcome = OUTCOME_V2_NAMESPACE; // required to be here
+
+/**
+ * @brief outcome::result<T> is a type which can be used as "value or error" type.
+ *
+ * @example see {@file test/deps/outcome_test.cpp}
+ *
+ * @code
+ * enum ConversionErrc {
```

Comment:
Errc means error code, though it contains Success code, maybe ConversionResultCode looks better.

Reply:
Technically, name does not matter. I put errc here just because original example has it.